docusaurus/docs/iOS/components/message-composer.md

```diff
@@ -135,18 +130,18 @@ Complete list of all the subviews that make the `ComposerView`.
 
 ## Composer Content
 
-The `ComposerVC.Content` is a struct that contains all the data that will be part of the composed message. It contains the current `text` of the message, the `attachments`, the `threadMessage` in case you are inside a Thread, the `command` if you are sending for example a Giphy, and the `state` of the composer to determine whether you are creating, editing or quoting a message.
+The `ComposerVC.Content` is a struct that contains all the data that will be part of the composed message. It contains the current `text` of the message, the `attachments`, the `threadMessage` in case you are inside a Thread, the `command` if you are sending, for example, a Giphy, and the `state` of the composer to determine whether you are creating, editing or quoting a message.
 
-Some of the composer's content properties are mutable, like the `attachments`, `threadMessage`, `text` and `command` properties. They can be directly changed since they represent data that do not depend on the state of the composer. On the other hand, there are properties that are immutable, and only can be changed through mutating functions on the `ComposerVC.Content`. This is to protect against bad states, for example, having the `editingMessage` property to `nil` but the `state = .edit`.
+Some of the composer's content properties are mutable, like the `attachments`, `threadMessage`, `text`, and `command` properties. They can be directly changed since they represent data that do not depend on the state of the composer. On the other hand, some properties are immutable, and can only be changed through mutating functions on the `ComposerVC.Content`. This is to protect against bad states, for example, having the `editingMessage` property to `nil` but the `state = .edit`.
```

Comment:
```suggestion
Some of the composer's content properties are mutable, like the `attachments`, `threadMessage`, `text`, and `command`. They can be directly changed since they represent data that do not depend on the state of the composer. On the other hand, some properties are immutable, and can only be changed through mutating functions on the `ComposerVC.Content`. This protects a composer against bad states, for example, having the `editingMessage` property to `nil` but the `state = .edit`.
```